src/Create/index.css

```diff
@@ -0,0 +1,17 @@
+.fade-enter {
+  height: 0px;
+}
+
+.fade-enter-active {
+  height: 60px;
+  transition: 100ms ease-in all;
+}
+
+.fade-exit {
```

Comment:
Is this intentionally empty?

Reply:
I left it there in case we want to add anything.  Right now it's just using it's current state.  In some examples you would add like opacity or whatever.

We could comment it out.

Reply:
Also as far as the height I just put 60px as an estimate, however, if there a specific height for each address we should use that.  I couldn't find it in the code.

Reply:
I'm pretty sure it's all percentages so feel free to pick something and then play around with it. I'm going to merge this